templates/class.mustache

```diff
@@ -2,10 +2,10 @@
 /// <reference path="{{&.}}" />
 {{/imports}}
 
-import request, { SuperAgentStatic, SuperAgentRequest, Response } from "superagent";
+import request, { SuperAgentStatic, SuperAgentRequest, Response, } from "superagent";
```

Comment:
Why the changes to the two template files? Those changes are not related to tslintfix being run